Mail/Views/New Message/RecipientField.swift

```diff
@@ -51,33 +51,35 @@ struct RecipientField: View {
     }
 
     var body: some View {
-        VStack(spacing: 0) {
-            if !recipients.isEmpty {
-                RecipientsList(
-                    focusedField: _focusedField,
-                    recipients: $recipients,
-                    isCurrentFieldFocused: isCurrentFieldFocused,
-                    type: type
-                )
-            }
+        HStack {
```

Comment:
The default value of spacing is nil which means SwiftUI will space the value by the "best" value according to him.
Here, we want a 0 spacing.
```suggestion
        HStack(spacing: 0) {
```